libraries/rush-lib/src/logic/installManager/WorkspaceInstallManager.ts

```diff
@@ -433,4 +487,27 @@ export class WorkspaceInstallManager extends BaseInstallManager {
       }
     }
   }
+
+  /**
+   * Used when invoking pnpm rebuild for running deferred installation scripts.
+   */
+  protected pushRebuildArgs(args: string[]): void {
```

Comment:
"Rebuild" has a specific meaning in the current version of Rush. Can we use a different word here?

Reply:
We could call it `pushPnpmRebuildArgs()`

Reply:
That kinda makes it sound like we're rebuilding PNPM.

Can we just call this operation "postInstallScripts" everywhere? So this could be `pushPnpmPostInstallScriptsArgs`

Reply:
We're now calling them "installation scripts" because `postinstall` is just one of several lifecycle events that happen during installation (`preinstall`, `install`, `prepare`, etc).

That said, @iclanton these are CLI arguments for the actual shell command `pnpm rebuild`.  PNPM's naming could be more intuitive, but `rebuild` is the name of the command.  It doesn't seem helpful for our source code to refer to it using a different name.

We could qualify it a bit as `pushPnpmRebuildCommandArgs`... and maybe add a code comment:

```ts
 /**
   * Used when invoking the "pnpm rebuild" command that runs installation scripts of NPM dependencies.
   * NOTE: Despite the name, "pnpm rebuild" is purely an installation action -- it does not build Rush projects
   * in the manner of "rush rebuild" or "rush build".
   */
  protected pushPnpmRebuildCommandArgs(args: string[]): void {
```